File: Backend/Models/Application.cs
using Backend.Dtos;
using Backend.Models.Enums;

namespace Backend.Models
{
    public class Applications
    {
        public int Id { get; set; }
        public string FirstName{ get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public EducationLevel EducationLevel { get; set; }
        public string CoverLetter { get; set; }
        public string CV { get; set; }
        public Status Status { get; set; }
        public ICollection<Comment>? Comments  { get; set; }
        public ICollection<Selection>? Selections { get; set; }



    }
}

File: Frontend/src/app/models/application.ts
import { educationLevels } from "./educationLevelEnum";
import { ApplicationComment } from "./applicationComment.model";

export class Application {
    id?: number;
    firstName: string;
    lastName: string;
    email: string;
    educationLevel: string;
    status: string;
    coverLetter: string;
    cv: string;
    comments: ApplicationComment[];
}